docs/project/roadmap.md

```diff
@@ -10,120 +10,108 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 <!-- tocstop -->
 
-## Objective for 2023: get ready to evaluate the Carbon Language.
+## Objective for 2024: a working toolchain that supports C++ interop
 
-Our focus throughout 2023 will be to get the Carbon Language and project ready
-for serious evaluation of the experiment. There are two aspects to this
-evaluation:
+Our focus for 2024 will be to get the Carbon toolchain working, including C++
+interop. We see three key criteria:
 
-1. The language and tools need to be complete enough to evaluate.
-2. The users and communities we are targeting need both context and awareness of
-   the technical ideas and design principles on which Carbon is built.
+-   Building realistic Carbon code for interesting interop with C++.
+-   Building realistic C++ code for interesting interop with Carbon.
+-   The interop itself to allow a single program mixing the two languages.
 
-## Key results in 2023
+This will both allow folks to explore Carbon using a more traditional and
+realistic compiler model, and allow that exploratory Carbon to lean on C++ for
+libraries and other functionality that doesn't exist in Carbon yet. It will also
+demonstrate how the interop will work in practice.
 
-### A concrete definition of our Minimum Viable Product for evaluation, the 0.1 language
+This objective and focus are oriented around the toolchain and implementation of
+Carbon. We still expect some work on language design, but for its priority to be
+driven largely as a function of being in the critical path of some aspect of our
+implementation work.
 
-While we have talked about our 0.1 language, or our Minimum Viable Product (MVP)
-for evaluation purposes, we need to pin down exactly what this includes. We need
-concrete milestones that need to be reached for us to have confidence in
-potential users and communities being able to evaluate Carbon as a successor to
-C++.
+## Key results in 2024
 
-We expect this to include a reasonably precise set of requirements across:
+### Carbon's toolchain implements the language enough to support interop with C++
 
--   Necessary language features
--   Nice-to-have features
--   Features we can omit and reasonably expect to not obstruct credible
-    evaluation
--   Implementation coverage of those features in the Carbon Explorer to validate
-    the design
--   Implementation coverage of those features in the toolchain
--   Quality of implementation in the toolchain, both in general and for specific
-    features
--   Necessary documentation, strategy, or other supporting material
+Some example language features that we think are key to success here but far
+from an exhaustive list:
 
-We expect this to also include at least enough of Carbon's features and C++
-interop to support [our basic example](/docs/images/snippets.md#mixed).
+-   Imports
+-   Operator overloading and dispatch for expressions
+-   Generic types and functions
+-   Templates (at least enough for interop)
```

Comment:
To be sure, if we don't implement any interop that would need template support this year, does that mean no template support?

Note though, template support might have substantial cost, even minimal template support.

Reply:
Sure, I would prioritize template support here pretty specifically at the same level as template support in interop. If we aren't getting towards wanting interop to support that, then it we also probably aren't going to need it on the Carbon side.